drivers/net/ethernet/renesas/rswitch.h

```diff
@@ -274,6 +274,13 @@ struct rswitch_filters {
 	DECLARE_BITMAP(cascade, PFL_CADF_N);
 };
 
+struct rswitch_vmq_status {
+	u64 front_tx, front_rx;
```

Comment:
It is good idea to introduce some `protocol_version` field, so we at least can refuse to work if backend and frontend use different protocols  versions.

Reply:
Added one more patch, that introduces this kind of enum.